python/doc/examples/reliability_sensitivity/design_of_experiments/plot_compute_ell2_error.py

```diff
@@ -0,0 +1,29 @@
+"""
+Compute the L2 error between two functions
+==========================================
+"""
+# %%
+
+# %%
+# In this example we compute the L2 error between two functions.
+
+# %%
+import openturns as ot
+import openturns.experimental as otexp
+from openturns.usecases import ishigami_function
+
+# %%
+# This is easy using the arithmetic operator `-` on two
+# instances of the :class:`~openturns.Function` class.
+
+# %%
+im = ishigami_function.IshigamiModel()
+partOfIshigamiFunction = ot.SymbolicFunction(
+    ['x1', 'x2', 'x3'], ['7 * (sin(x2))^2 + 0.1 * x3^4 * sin(x1)']
+)
+ot.RandomGenerator.SetSeed(0)
+sampleSize = 32768
```

Comment:
This value, $2^{15}$ was chosen with the goal of having the smallest sample size (so that the example is fast) such that the output has 2 significant digits. The original content was inside an API example for which the output is check at build time. This search was based on a base-2 sample size increase. I can set a more ordinary value now.

Reply:
Fixed at 354dcd965.